styles/footer.css

```diff
@@ -0,0 +1,101 @@
+*{
+    margin: 0%;
+}
+
+.footer-container{
+    background-color:#26272b;
+    padding:1% 0 0;
+    font-size:1rem;
+    height:auto;
+    color:#737373;
+    display: flex;
+}
+.directory-container{
+    height:100%;
+    width: 25%;
+
+}
+    .directory-img{
+        width: 200px;
+        padding:0 20px 0;
+
+    }
+    .directory-info{
+        color:white;
+        height: 70%;
+        padding:15px;
+        color:#737373;
+    }
+.about-us{
+    height: 100%;
+    width: 25%;
+    text-align: center;
+}
+    .about-us-title{
+        height: 30%;
+        color: white;
+        text-align: center;
+        margin:0;
+        
+    }
+    .about-us-info{
+        height: 70%;
+        color: white;
+        padding: 15px;
+        color:#737373;
+
+    }
+
+.colaboradores-container{
+    height: 100%;
+    width: 25%;
+}
+
+    .colaboradores-title{
+        height: 30%;
+        color: white;
+        text-align: center;
+        margin:0;
+
+    }
+    .link-container{
+        height: 70%;
+        padding: 15px;
+        text-align: center;
+        display: grid;
+        
+    
+    }
+        .link-web-page{
+            color:#737373;
+            width:100%;
+            text-decoration: none;
+        }
+            .link-web-page:hover{
+                color:rgba(106, 59, 89, 1);
```

Comment:
do not use rgba if you're not using the opacity feature

Reply:
fixed !